### ISSUE TITLE
glance_image ignores min_ram and min_disk parameters

### ISSUE THREAD

**Issue by ericwb** [ansible/ansible-modules-core#1424]
The documentation for the glance_image module states you can pass min_ram and min_disk, but in fact neither of these options are used.  

Search for 'min_ram' and you'll notice only two places (in doc section and to read from args).
https://github.com/ansible/ansible-modules-core/blob/devel/cloud/openstack/glance_image.py

**Comment by ansibot** [comment]
@emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by rcarrillocruz** [comment]
Hi @ericwb 

glance_image is deprecated, you should use os_image instead.

Please give it a try and let us know if you have this issue (from what I read in the code of os_image min_ram and min_disk is indeed used).

Thanks

**Comment by Shrews** [comment]
os_image does support these params. Please use the newer modules.

wontfix
